cmd/insprd/memory/brokers/factory.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/inspr/inspr/pkg/ierrors"
 	"github.com/inspr/inspr/pkg/meta"
 	"github.com/inspr/inspr/pkg/operator/k8s"
-	"github.com/inspr/inspr/pkg/sidecar/models"
+	"github.com/inspr/inspr/pkg/sidecar_old/models"
```

Comment:
I hate this, no two word modules, ugly ugly ugly

Reply:
This will be removed. It was only maintained whilst the Kafka-specific sidecar is developed